### PR TITLE
Add implementation of standard deviation on data

### PR DESCRIPTION
**Pull request by rk-8194**
Extend the graphs to show standard deviation according to SR1.1.1.

### CODE REVIEW

File: inflammation/models.py [hunk 1]
```diff
@@ -54,3 +54,13 @@ def daily_min(data):
     """Calculate the daily min of a 2d inflammation data array."""
     return np.min(data, axis=0)
 
+
+def s_dev(data):
+    """Computes and returns standard deviation for data."""
+    mmm = np.mean(data, axis=0)
```
Comment (rk-8194) [inline inflammation/models.py:60]: Rename variable to something like 'mean'.

File: inflammation/models.py [hunk 2]
```diff
@@ -54,3 +54,13 @@ def daily_min(data):
...
+    devs = []
+    for entry in data:
+        devs.append((entry - mmm) * (entry - mmm))
+
+    s_dev2 = sum(devs) / len(data)
```
Comment (rk-8194) [inline inflammation/models.py:65]: Rename to s_dev.

File: inflammation/models.py [hunk 3]
```diff
@@ -54,3 +54,13 @@ def daily_min(data):
     """Calculate the daily min of a 2d inflammation data array."""
     return np.min(data, axis=0)
 
+
+def s_dev(data):
```
Comment (rk-8194) [inline inflammation/models.py:58]: Rename to something like compute_standard_deviation.

**Review by JBello1610 — COMMENTED** [review]
Except for minor naming changes which have already been raised in previous comments, this looks good!

**Review by JBello1610 — CHANGES_REQUESTED** [review]
Looks good, please add the following tests:

 - Test for the elements in the view_data dictionary with `average`, ` max`, `min` and optional for the output of `s_dev` function
 -  Test the outputs of `s_dev` function for different cases (including edge cases).
 - Test the loop in `s_dev()` for empty, single and multielement sequences.
 - Test the `test_daily_standard_deviation()` function

**Comment by rk-8194** [comment]
Raised new tests as issue.
